specification/storagecache/resource-manager/Microsoft.StorageCache/stable/2020-03-01/storagecache.json

```diff
@@ -0,0 +1,1776 @@
+            "description": "Cache deleted.",
+            "schema": {}
+          },
+          "202": {
+            "description": "Started the Cache's transition to Deleted state. Poll the Cache to monitor.",
+            "schema": {}
+          },
+          "204": {
+            "description": "Cache deleted.",
+            "schema": {}
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Caches_Delete": {
+            "$ref": "./examples/Caches_Delete.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "tags": [
+          "Caches"
+        ],
+        "description": "Schedules a Cache for deletion.",
+        "operationId": "Caches_Delete"
+      },
+      "get": {
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "description": "Target resource group.",
+            "in": "path",
+            "name": "resourceGroupName",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "description": "Name of Cache.",
+            "in": "path",
+            "name": "cacheName",
+            "pattern": "^[-0-9a-zA-Z_]{1,31}$",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Returns the Cache object corresponding to cacheName.",
+            "schema": {
+              "$ref": "#/definitions/Cache"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Caches_Get": {
+            "$ref": "./examples/Caches_Get.json"
+          }
+        },
+        "tags": [
+          "Caches"
+        ],
+        "description": "Returns a Cache.",
+        "operationId": "Caches_Get"
+      },
+      "put": {
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "description": "Target resource group.",
+            "in": "path",
+            "name": "resourceGroupName",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "description": "Name of Cache.",
```

Comment:
It will be quite helpful to end users to add name constraints into description also.

Reply:
Add name constraints to description in next commit.